scripts/enable-ecs-agent-inferentia-support.sh

```diff
@@ -31,10 +31,7 @@ sudo yum install -y aws-neuronx-dkms-2.*
 sudo yum install -y aws-neuronx-oci-hook-2.*
 
 # Install oci-add-hooks
-# TODO: oci-add-hooks package has compatibility issue with AL2023 IMDSv2. Remove condition after root caused and resolved
```

Comment:
The original author of this TODO nor the reviewers recall what exactly this compatibility issue was from 2022. My testing did not encounter such an issue.